filter/etc/e-smith/templates/etc/rspamd/local.d/actions.conf/10base

```diff
@@ -4,18 +4,20 @@
 {
 my $reject  = $rspamd{SpamKillLevel} || '15';
 my $add     = $rspamd{SpamTag2Level} || '6';
-my $rewrite = $rspamd{SpamTag2Level} || '6';
+my $rewrite = $add + 0.01;
```

Comment:
Use float only if conversion to string does not use scientific notation. As alternative, just append ".01" to the string value

Reply:
...or (better) use `sprintf()`

Reply:
@DavidePrincipi we cannot just append '.01' because what will occur when the `SpamTag2Level` equals `9.1`, we will have `9.1.01`